package.json

```diff
@@ -45,6 +45,7 @@
     "near-api-js": "^0.30.0",
     "near-seed-phrase": "^0.1.0",
     "open": "^7.0.1",
+    "prompts": "^2.3.2",
```

Comment:
we are using `readline` in other places when prompting user. I think we either need to update everything to prompts or continue using `readline`

Reply:
Using readline and chalk now

Reply:
change is still here though

Reply:
I think it was cached? It's not in `package.json` anymore